backend/common/src/test/java/ai/verta/modeldb/common/futures/FutureTest.java

```diff
@@ -144,6 +145,57 @@ void retry() throws Exception {
         .isEqualTo("success!");
   }
 
+  @Test
+  void retry_strategy_failure() {
+    AtomicInteger sequencer = new AtomicInteger();
+
+    assertThatThrownBy(
+            () ->
+                Future.retrying(
+                        () -> {
+                          if (sequencer.getAndIncrement() < 10) {
```

Comment:
Is the goal of the # 10 here to just be more than 6 (the expected max number)? Or is it more specific? If the goal here is just to be "too many retries" maybe pull the literal into a variable named `tooManyRetries` or something?

Reply:
yeah, it's just intended to be more times than the maximum configured. I'll rename it. good suggestion!